specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf

```diff
@@ -242,6 +242,15 @@
             ],
             "fragmentShader": "CesiumTexturedBoxTest0FS",
             "vertexShader": "CesiumTexturedBoxTest0VS"
+        },
+        "unusedProgramId": {
+            "attributes": [
```

Comment:
Throughout this PR, can we simplify this a bit, e.g., instead of
```
"attributes": [
    "a_normal",
    "a_position",
    "a_texcoord0"
]
```
Just use
```
"attributes": [
    "a_position"
]
```